vite.config.js

```diff
@@ -66,19 +78,21 @@ const esm = defineConfig({
 });
 
 const editorConfig = defineConfig({
```

Comment:
Perhaps, we can change this to something like `dev` to make it clear that this config is used when developing either Editor or `embed`.

```suggestion
const dev = defineConfig({
```